LICENCE

```diff
@@ -6,24 +6,50 @@ in the prover_snapshots directory (which include copies of their
 licences), is subject to the BSD two-clause licence below.
 
 Copyright (c) 2017-2023
-  Prashanth Mundkur
-  Rishiyur S. Nikhil and Bluespec, Inc.
-  Jon French
-  Brian Campbell
-  Robert Norton-Wright
-  Alasdair Armstrong
-  Thomas Bauereiss
-  Shaked Flur
-  Christopher Pulte
-  Peter Sewell
-  Alexander Richardson
-  Hesham Almatary
-  Jessica Clarke
-  Microsoft, for contributions by Robert Norton-Wright and Nathaniel Wesley Filardo
-  Peter Rugg
-  Aril Computer Corp., for contributions by Scott Johnson
-  Philipp Tomsich
-  VRULL GmbH, for contributions by its employees
+   ahadali5000
+   Alasdair Armstrong
+   Alex Richardson
+   Alexander Richardson
+   Aril Computer Corp., for contributions by Scott Johnson
+   Ben Marshall
+   Bicheng Yang
+   Bilal Sakhawat
+   Brian Campbell
+   Chris Casinghino
+   Christopher Pulte
+   Codasip, for contributions by Tim Hutt
+   dylux
+   eroom1966
+   Hesham Almatary
+   James Clarke
+   Jan Henrik Weinstock
+   Jessica Clarke
+   Jon French
+   Martin Berger
+   Michael Sammler
+   Microsoft, for contributions by Robert Norton-Wright and Nathaniel Wesley Filardo
+   Muhammad Bilal Sakhawat
+   Nathaniel Wesley Filardo
+   Paul A. Clarke
+   Peter Rugg
+   Peter Sewell
+   PeterRugg
+   Philipp Tomsich
+   Prashanth Mundkur
+   Rafael Sene
+   Rishiyur S. Nikhil (Bluespec, Inc.)
+   Robert Norton-Wright
+   Scott Johnson (Aril Inc.)
+   Shaked Flur
+   Thibaut Pérami
+   Thomas Bauereiss
+   VRULL GmbH, for contributions by its employees
+   William McSpaddden
+   Xinlai Wan
```

Comment:
This list is going to get stale at some point, is it expected that every new contribution also updates this list?

Reply:
Yeah it didn't look like it had been kept up to date so far, which is why I added the note about `git shortlog`. However I didn't want to just delete it because it contains some information not in `git shortlog`, e.g. company associations and some people's real names.

I think the only real reason to keep a list like this is to make relicensing easier if it ever came to that, but that seems extremely unlikely given the license is BSD. So I think updating it for new contributors is probably "nice to have but doesn't really matter".